.travis.yml

```diff
@@ -31,9 +31,11 @@ install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git $HOME/maintainer-quality-tools
   - export PATH=$HOME/maintainer-quality-tools/travis:$PATH
   - travis_install_nightly
+  - pip install pytz
   - git clone https://github.com/OCA/sale-workflow $HOME/sale-workflow -b ${VERSION}
   - git clone https://github.com/OCA/bank-statement-reconcile $HOME/bank-statement-reconcile -b ${VERSION}
   - git clone https://github.com/OCA/product-attribute $HOME/product-attribute -b ${VERSION}
+  - git clone https://github.com/OCA/website $HOME/website -b ${VERSION}
```

Comment:
Please take the occasion to change all of these dependencies to the oca_dependencies.txt file (it only needs one line per repo with its short name - website, product-attribute... -)